meta-lxatac-bsp/recipes-kernel/linux/files/defconfig

```diff
@@ -15,7 +15,7 @@ CONFIG_CC_HAS_ASM_GOTO_OUTPUT=y
 CONFIG_CC_HAS_ASM_GOTO_TIED_OUTPUT=y
 CONFIG_CC_HAS_ASM_INLINE=y
 CONFIG_CC_HAS_NO_PROFILE_FN_ATTR=y
-CONFIG_PAHOLE_VERSION=120
+CONFIG_PAHOLE_VERSION=124
```

Comment:
This is an annoying hunk, as it only contains the version of pahole you have installed. IMHO this should not be a kconfig item, but a make runtime variable. (Nothing to blame on you.)